data.sql

```diff
@@ -0,0 +1,6 @@
+/* Populate the database with sample data. */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 0, false, 15.04);
```

Comment:
- [ ] Please correct this query as below because the escape_attempts are 1 not 0.
`INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Pikachu', '2021-01-07', '1', 'false', '15.04');
`

Reply:
well resolved! ✔️